server/runtime_nakama_module.go

```diff
@@ -298,6 +299,109 @@ func (n *NakamaModule) usersFetchHandle(l *lua.LState) int {
 	return 1
 }
 
+func (n *NakamaModule) usersUpdate(l *lua.LState) int {
+	updatesTable := l.CheckTable(1)
+	if updatesTable == nil || updatesTable.Len() == 0 {
+		l.ArgError(1, "expects a valid set of user updates")
+		return 0
+	}
+
+	conversionError := ""
+	updates := make([]*SelfUpdateOp, 0)
+
+	updatesTable.ForEach(func(i lua.LValue, u lua.LValue) {
+		updateTable, ok := u.(*lua.LTable)
+		if !ok {
+			conversionError = "expects a valid set of user updates"
+			return
+		}
+
+		updateTable.ForEach(func(k lua.LValue, v lua.LValue) {
+			update := &SelfUpdateOp{}
+			switch k.String() {
+			case "UserId":
+				if v.Type() != lua.LTString {
+					conversionError = "expects valid user IDs in each update"
+					return
+				}
+				if uid, err := uuid.FromString(v.String()); err != nil {
+					conversionError = "expects valid user IDs in each update"
+					return
+				} else {
+					update.UserId = uid.Bytes()
+				}
+			case "Handle":
+				if v.Type() != lua.LTString {
+					conversionError = "expects valid handles in each update, when provided"
```

Comment:
Let's remove the `, when provided` suffix. This error only occurs when they have provided the Handle, can't occur in any other condition.